bindings/zeebe/command/deploy_process.go

```diff
@@ -38,10 +38,9 @@ func (z *ZeebeCommand) deployProcess(ctx context.Context, req *bindings.InvokeRe
 		return nil, ErrMissingFileName
 	}
 
-	// TODO: Replace with a method that is not deprecated.
-	response, err := z.client.NewDeployProcessCommand(). //nolint:staticcheck
-								AddResource(req.Data, deployFileName).
-								Send(ctx)
+	response, err := z.client.NewDeployResourceCommand().
+		AddResource(req.Data, deployFileName).
```

Comment:
The linter wasn't happy with `//nolint:staticcheck` and wasn't happy without it either. Rather than fighting the linter (lesigh) I just replaced this with the non-deprecated method.